docs/configuration.md

```diff
@@ -2309,6 +2309,16 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>0.6.0</td>
 </tr>
+<tr>
+  <td><code>spark.speculation.min.threshold</code></td>
+  <td>100ms</td>
+  <td>
+    Duplicate copies of a task will only be launched if the original copy has been running for
```

Comment:
I think we should reword a bit to be more specific about the config. something like:  Minimum amount of time a task runs before being considered for speculation. This ....

Reply:
done